bin/cli.js

```diff
@@ -3,17 +3,17 @@
 'use strict';
 
 const minimist = require('minimist');
-const _ = require('lodash');
+const camelCase = require('lodash.camelcase');
 const checkDependencies = require('../lib/check-dependencies');
 
 const argv = minimist(process.argv.slice(2));
 
 // camelCase the options
-for (const key in argv) {
+Object.keys(argv).forEach(key => {
```

Comment:
I agree on "reads better". On performance: theoretically, it should be faster (because no `[[HasProperty]]` checks and function calls), but it is not yet optimized by V8 as good as good old `forEach`. Changed to `for of`.